libnm-core/nm-setting-wireless.h

```diff
@@ -57,6 +99,8 @@ G_BEGIN_DECLS
 #define NM_SETTING_WIRELESS_HIDDEN      "hidden"
 #define NM_SETTING_WIRELESS_POWERSAVE   "powersave"
 #define NM_SETTING_WIRELESS_MAC_ADDRESS_RANDOMIZATION   "mac-address-randomization"
+#define NM_SETTING_WIRELESS_WAKE_ON_WLAN "wake-on-wlan"
+#define NM_SETTING_WIRELESS_WAKE_ON_WLAN_PASSWORD "wake-on-wlan-password"
```

Comment:
this name is a bit unfortunate, because it has the same prefix as `NMSettingWirelessWakeOnWLan` flags. Maybe the enum should be called `NMWirelessWakeOnWLanFlags`, and the values `NM_WIRELESS_WAKE_ON_WLAN_FLAGS_*`. What do others thing?